Dockerfile

```diff
@@ -10,6 +10,8 @@ WORKDIR /usr/src/poker
 COPY Pipfile Pipfile
 COPY Pipfile.lock Pipfile.lock
 RUN pipenv install --system
+COPY nginx.conf /etc/nginx/sites-enabled/poker
```

Comment:
copy the file to `/etc/nginx/sites-available` and create a symlink in `/etc/nginx/sites-enabled`

Reply:
done